docs/explanation/api-and-clients.md

```diff
@@ -55,4 +63,15 @@ identities:
             user-id: 42
 ```
 
+An example identity named "alice" with `basic` type and `metrics` access level is shown below:
+
+```yaml
+identities:
+    alice:
+        access: metrics
+        basic:
+            # The password is sha512-hashed.
```

Comment:
It's actually hashed using the sha512-crypt algorithm, which is a bunch of rotations and salt and stuff on top of bare sha512:
```suggestion
            # The password is hashed using sha512-crypt, as generated by "openssl passwd -6".
```